src/main/webapp/app/exercises/modeling/assess/modeling-assessment-editor/modeling-assessment-dashboard.component.html

```diff
@@ -5,7 +5,7 @@ <h4>
     <jhi-assessment-filters (filterChange)="updateFilteredSubmissions($event)" [submissions]="submissions"></jhi-assessment-filters>
     <jhi-assessment-warning [exercise]="exercise" [submissions]="filteredSubmissions"></jhi-assessment-warning>
     <h4>
-        <small [ngStyle]="assessedSubmissions === submissions.length && submissions.length > 0 ? { color: '#5cb85c' } : {}">
+        <small [ngStyle]="assessedSubmissions === submissions.length && submissions.length > 0 ? { color: '#5cb85c' } : { color: '#d86b1f' }">
```

Comment:
Please do not hardcode colors. We have different themes, and make sure it looks good in both themes. I know it was like this before, but from now on we need to look out for this! :)

You can probably just replace the green color with ``var(--success)`` and the orange color with  ``var(--warning)``. 

If not, and you need custom colors, please add variables to ``src/main/webapp/content/scss/themes/_default-variables.scss`` and ``_dark_variables.scss``, and choose good colors for both themes. Then use that variable again with ``var(-- ...)``.

Additionally, we should conditionally add a class to this element rather than using inline styles.

Reply:
Sorry. I did not know. I will modify it accordenly. THank you!